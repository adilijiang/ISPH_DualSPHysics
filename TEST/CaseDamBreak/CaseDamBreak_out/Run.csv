#RunName;RunCode;DateTime;Np;TSimul;TSeg;TTotal;MemCpu;MemGpu;Steps;PartFiles;PartsOut;MaxParticles;MaxCells;Hw;StepAlgo;Kernel;Viscosity;ViscoValue;DeltaSPH;TMax;Nbound;Nfixed;H;RhopOut;PartsRhopOut;PartsVelOut;CellMode;RunMode;VA-Init;NL-Limits;NL-MakeSort;NL-SortData;NL-OutCheck;CF-PreForces;CF-Forces;SU-Shifting;SU-ComputeStep;SU-Floating;SU-Motion;SU-Periodic;SU-ResizeNp;SU-SavePart
CaseDamBreak;kx6hrmqh;04-11-2015 12:18:21;21001;2301.828613;460.363800;2301.850342;4865128;0;161979;101;165;21001;22720;Cpu;Symplectic;Wendland;Laminar+SPS;0.020000;0;5.000000;1001;1001;0.014142;(700-1300);0;0;2H;Pos-Simple, OpenMP(Threads:8);0.021895;52.482224;80.425407;144.068115;0.010804;268.176331;1483.296265;0.000000;272.350555;0.000000;0.000000;0.000000;0.000000;0.744807
