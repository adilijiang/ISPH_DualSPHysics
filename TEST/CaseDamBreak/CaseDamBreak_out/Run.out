

<DUALSPHYSICS> Copyright (C) 2015 by Dr Jose M. Dominguez, Dr Alejandro Crespo,
Prof. M. Gomez Gesteira, Dr Anxo Barreiro, Dr Ricardo Canelas, Dr C. Altomare
Dr Benedict Rogers, Dr Georgios Fourtakas, Dr Athanasios Mokos,
Dr Stephen Longshaw, Dr Renato Vacondio

EPHYSLAB Environmental Physics Laboratory, Universidade de Vigo
School of Mechanical, Aerospace and Civil Engineering, University of Manchester

DualSPHysics is free software: you can redistribute it and/or
modify it under the terms of the GNU General Public License as
published by the Free Software Foundation, either version 3 of
the License, or (at your option) any later version.

DualSPHysics is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License,
along with DualSPHysics. If not, see <http://www.gnu.org/licenses/>.


DualSPHysics Re v4.0.015 (03-10-2015)
======================================
Threads by host for parallel execution: 8
[Initialising JSphCpuSingle v0.70  04-11-2015 12:18:21]

*** Attention: Eps value is not used (this correction is deprecated).

**Basic case configuration is loaded
**Special case configuration is loaded
Loading initial state of particles...
Loaded particles: 21001
MapRealPos(border)=(-1.41421e-005,-1.41421e-005,-1.41421e-005)-(4.00001,1.41421e-005,3.00001)
MapRealPos(final)=(-1.41421e-005,-1.41421e-005,-1.41421e-005)-(4.00001,1.41421e-005,4.50003)
**Initial state of particles is loaded
**2D-Simulation parameters:
CaseName="CaseDamBreak"
RunName="CaseDamBreak"
SvDouble=False
SvTimers=True
SvTimersStep=0.000000
StepAlgorithm="Symplectic"
Kernel="Wendland"
Viscosity="Laminar+SPS"
Visco=0.020000
ViscoBoundFactor=1.000000
DeltaSph="None"
Shifting="None"
RenCorrection=0.000000
FloatingFormulation="None"
FloatingCount=0
CaseNp=21001
CaseNbound=1001
CaseNfixed=1001
CaseNmoving=0
CaseNfloat=0
CaseNfluid=20000
PeriodicActive=0
Dx=0.01
H=0.014142
CoefficientH=1
CteB=1115537.125000
Gamma=7.000000
RhopZero=1000.000000
Eps=0
Cs0=88.3672
CFLnumber=0.200000
DtIni=0.0001
DtMin=1e-005
DtAllParticles=False
MassFluid=0.100000
MassBound=0.100000
Bwen (wendland)=-984716.812500
SpsSmag=0.000001
SpsBlin=0.000000
TimeMax=5
TimePart=0.05
Gravity=(0.000000,0.000000,-9.810000)
NpMinimum=1001
RhopOut=True
RhopOutMin=700.000000
RhopOutMax=1300.000000
**Requested cpu memory for 21001 particles: 3.5 MB.
CellOrder="XYZ"
CellMode="2H"
Hdiv=1
MapCells=(142,1,160)
DomCells=(142,1,160)
DomCellCode="13_7_12"
**CellDiv: Requested cpu memory for 22051 particles: 0.7 MB.
**CellDiv: Requested cpu memory for 5548 cells (CellMode=2H): 0.1 MB.
RunMode="Pos-Simple, OpenMP(Threads:8)"
Allocated memory in CPU: 4490620 (4.28 MB)
Part_0000        21001 particles successfully stored

[Initialising simulation (kx6hrmqh)  04-11-2015 12:18:21]
PART       PartTime      TotalSteps    Steps    Time/Seg   Finish time        
=========  ============  ============  =======  =========  ===================
Part_0001      0.050012          1565     1565     411.97  04-11-2015 12:52:40
Part_0002      0.100010          3136     1571     423.83  04-11-2015 12:53:09
Part_0003      0.150024          4711     1575     437.40  04-11-2015 12:53:42
Part_0004      0.200012          6290     1579     446.70  04-11-2015 12:54:10
Part_0005      0.250025          7874     1584     451.82  04-11-2015 12:54:32
Part_0006      0.300001          9460     1586     449.58  04-11-2015 12:54:45
Part_0007      0.350006         11048     1588     453.32  04-11-2015 12:54:56
Part_0008      0.400003         12638     1590     453.07  04-11-2015 12:55:05
**CellDiv: Requested cpu memory for 11466 cells (CellMode=2H): 0.2 MB.
Part_0009      0.450016         14229     1591     452.25  04-11-2015 12:55:12
Part_0010      0.500021         15821     1592     453.17  04-11-2015 12:55:16
Part_0011      0.550026         17414     1593     453.66  04-11-2015 12:55:21
Part_0012      0.600002         19007     1593     453.82  04-11-2015 12:55:26
Part_0013      0.650002         20602     1595     453.98  04-11-2015 12:55:28
Part_0014      0.700018         22198     1596     454.98  04-11-2015 12:55:32
Part_0015      0.750018         24486     2288     653.28  04-11-2015 12:56:41
Part_0016      0.800008         26737     2251     644.31  04-11-2015 12:57:39
Part_0017      0.850002         28885     2148     622.23  04-11-2015 12:58:23
**CellDiv: Requested cpu memory for 23452 cells (CellMode=2H): 0.4 MB.
Part_0018      0.900014         30941     2056     587.66  04-11-2015 12:58:53
Part_0019      0.950016         32907     1966     561.35  04-11-2015 12:59:13
Part_0020      1.000008         34785     1878     537.50  04-11-2015 12:59:24
Part_0021      1.050012         36576     1791     511.12  04-11-2015 12:59:28
Part_0022      1.100011         38273     1697     484.82  04-11-2015 12:59:26
Part_0023      1.150009         39870     1597     457.88  04-11-2015 12:59:19
  Particles out: 3  (total: 3)
Part_0024      1.200009         41451     1581     450.65  04-11-2015 12:59:11
  Particles out: 13  (total: 16)
Part_0025      1.250006         43029     1578     449.97  04-11-2015 12:59:02
  Particles out: 31  (total: 47)
Part_0026      1.300011         44603     1574     447.60  04-11-2015 12:58:55
  Particles out: 28  (total: 75)
Part_0027      1.350002         46176     1573     447.46  04-11-2015 12:58:47
  Particles out: 32  (total: 107)
Part_0028      1.400021         47749     1573     446.01  04-11-2015 12:58:40
  Particles out: 40  (total: 147)
Part_0029      1.450006         49320     1571     447.54  04-11-2015 12:58:34
  Particles out: 18  (total: 165)
Part_0030      1.500014         50892     1572     443.96  04-11-2015 12:58:28
Part_0031      1.550027         52464     1572     456.15  04-11-2015 12:58:24
Part_0032      1.600024         54035     1571     452.86  04-11-2015 12:58:19
Part_0033      1.650023         55606     1571     445.60  04-11-2015 12:58:14
Part_0034      1.700002         57178     1572     444.71  04-11-2015 12:58:09
Part_0035      1.750005         58752     1574     445.92  04-11-2015 12:58:04
Part_0036      1.800010         60328     1576     446.42  04-11-2015 12:58:00
Part_0037      1.850029         61907     1579     453.82  04-11-2015 12:57:57
Part_0038      1.900009         63487     1580     454.52  04-11-2015 12:57:54
Part_0039      1.950016         65069     1582     447.53  04-11-2015 12:57:51
Part_0040      2.000014         66683     1614     457.02  04-11-2015 12:57:49
Part_0041      2.050000         68279     1596     454.70  04-11-2015 12:57:47
Part_0042      2.100009         69883     1604     458.74  04-11-2015 12:57:45
Part_0043      2.150028         71484     1601     452.68  04-11-2015 12:57:42
Part_0044      2.200006         73087     1603     453.68  04-11-2015 12:57:40
Part_0045      2.250008         74693     1606     452.91  04-11-2015 12:57:38
Part_0046      2.300009         76314     1621     457.39  04-11-2015 12:57:37
Part_0047      2.350015         77932     1618     455.43  04-11-2015 12:57:34
Part_0048      2.400029         79570     1638     461.54  04-11-2015 12:57:33
Part_0049      2.450012         81180     1610     454.99  04-11-2015 12:57:32
Part_0050      2.500025         82812     1632     458.72  04-11-2015 12:57:31
Part_0051      2.550026         84488     1676     471.69  04-11-2015 12:57:31
Part_0052      2.600024         86192     1704     480.66  04-11-2015 12:57:32
Part_0053      2.650003         87810     1618     458.83  04-11-2015 12:57:31
Part_0054      2.700016         89411     1601     453.67  04-11-2015 12:57:29
Part_0055      2.750020         91020     1609     461.97  04-11-2015 12:57:28
Part_0056      2.800004         92628     1608     457.06  04-11-2015 12:57:27
Part_0057      2.850006         94212     1584     450.05  04-11-2015 12:57:26
Part_0058      2.900003         95791     1579     448.82  04-11-2015 12:57:24
Part_0059      2.950000         97375     1584     450.12  04-11-2015 12:57:23
Part_0060      3.000026         98949     1574     446.52  04-11-2015 12:57:21
Part_0061      3.050017        100519     1570     452.92  04-11-2015 12:57:20
Part_0062      3.100026        102090     1571     449.48  04-11-2015 12:57:18
Part_0063      3.150017        103659     1569     444.56  04-11-2015 12:57:16
Part_0064      3.200025        105229     1570     444.76  04-11-2015 12:57:15
Part_0065      3.250024        106799     1570     444.79  04-11-2015 12:57:13
Part_0066      3.300013        108367     1568     449.45  04-11-2015 12:57:11
Part_0067      3.350023        109940     1573     446.80  04-11-2015 12:57:10
Part_0068      3.400023        111523     1583     452.13  04-11-2015 12:57:09
Part_0069      3.450006        113138     1615     460.54  04-11-2015 12:57:08
Part_0070      3.500015        114713     1575     452.85  04-11-2015 12:57:08
Part_0071      3.550001        116292     1579     454.89  04-11-2015 12:57:07
Part_0072      3.600008        117865     1573     458.58  04-11-2015 12:57:07
Part_0073      3.650018        119436     1571     448.76  04-11-2015 12:57:05
Part_0074      3.700007        121006     1570     448.52  04-11-2015 12:57:05
Part_0075      3.750007        122575     1569     446.19  04-11-2015 12:57:03
Part_0076      3.800014        124146     1571     447.18  04-11-2015 12:57:02
Part_0077      3.850015        125717     1571     445.08  04-11-2015 12:57:00
Part_0078      3.900000        127287     1570     448.32  04-11-2015 12:57:00
Part_0079      3.950027        128858     1571     449.56  04-11-2015 12:56:58
Part_0080      4.000006        130429     1571     445.76  04-11-2015 12:56:58
Part_0081      4.050006        132001     1572     447.89  04-11-2015 12:56:57
Part_0082      4.100028        133576     1575     452.81  04-11-2015 12:56:56
Part_0083      4.150027        135144     1568     450.27  04-11-2015 12:56:55
Part_0084      4.200003        136712     1568     446.19  04-11-2015 12:56:54
Part_0085      4.250022        138281     1569     447.09  04-11-2015 12:56:53
Part_0086      4.300001        139850     1569     449.56  04-11-2015 12:56:52
Part_0087      4.350022        141427     1577     447.11  04-11-2015 12:56:52
Part_0088      4.400017        143015     1588     449.71  04-11-2015 12:56:51
Part_0089      4.450004        144610     1595     450.49  04-11-2015 12:56:51
Part_0090      4.500023        146240     1630     459.99  04-11-2015 12:56:50
Part_0091      4.550019        147838     1598     455.39  04-11-2015 12:56:49
Part_0092      4.600024        149421     1583     446.81  04-11-2015 12:56:49
Part_0093      4.650009        150999     1578     446.09  04-11-2015 12:56:48
Part_0094      4.700022        152574     1575     445.14  04-11-2015 12:56:47
Part_0095      4.750025        154146     1572     444.54  04-11-2015 12:56:47
Part_0096      4.800013        155715     1569     444.57  04-11-2015 12:56:46
Part_0097      4.850030        157283     1568     443.85  04-11-2015 12:56:45
Part_0098      4.900026        158848     1565     442.73  04-11-2015 12:56:44
Part_0099      4.950005        160412     1564     442.93  04-11-2015 12:56:43
Part_0100      5.000021        161979     1567     444.06  04-11-2015 12:56:43

[Simulation finished  04-11-2015 12:56:43]
Particles of simulation (initial): 21001
DTs adjusted to DtMin............: 0
Excluded particles...............: 165
Total Runtime....................: 2301.850342 sec.
Simulation Runtime...............: 2301.828613 sec.
Time per second of simulation....: 460.363800 sec.
Steps per second.................: 70.369705
Steps of simulation..............: 161979
PART files.......................: 101
Maximum number of particles......: 21001
Maximum number of cells..........: 22720
CPU Memory.......................: 4865128 (4.64 MB)

[CPU Timers]
VA-Init..........................: 0.021895 sec.
NL-Limits........................: 52.482224 sec.
NL-MakeSort......................: 80.425407 sec.
NL-SortData......................: 144.068115 sec.
NL-OutCheck......................: 0.010804 sec.
CF-PreForces.....................: 268.176331 sec.
CF-Forces........................: 1483.296265 sec.
SU-Shifting......................: 0.000000 sec.
SU-ComputeStep...................: 272.350555 sec.
SU-Floating......................: 0.000000 sec.
SU-Motion........................: 0.000000 sec.
SU-Periodic......................: 0.000000 sec.
SU-ResizeNp......................: 0.000000 sec.
SU-SavePart......................: 0.744807 sec.
 
